manifests/init.pp

```diff
@@ -43,6 +43,16 @@
   $autostart_all = true,
 ) {
 
+  $etc_directory = $::operatingsystem ? {
```

Comment:
Could you move this directive into `openvpn::params`, wo we have these OS specific thing all at one place.